### ISSUE TITLE
Consider using `behavior=keep_as_draft` (not `behavoir: void`) when suspending WCPay Subscriptions

### ISSUE THREAD

**Issue by haszari** [Automattic/woocommerce-payments#5546]
### Describe the bug
We currently [set subscriptions as `void` in Stripe when suspending a subscription](https://github.com/Automattic/woocommerce-payments/blob/trunk/includes/subscriptions/class-wc-payments-subscription-service.php#L496-L509).

This means the related invoice cannot be collected, so if this happens (for whatever reason) the merchant has limited options for collecting payment. 

If we switch [`keep_as_draft`](https://stripe.com/docs/billing/subscriptions/pause#collect-payment-later), then it might be possible to re-invoice and recover payment even if the subscription was temporarily or accidentally suspended.

Note this has not been validated, the first step would be to prototype/explore this. But it looks like a good option.

### To Reproduce
TBC

### Additional context
Internal discussion: pdjTHR-27D-p2#comment-2717

**Comment by zmaglica** [comment]
@haszari is this issue on team Helix's radar? I am triaging the WC Pay backlog, and I am not certain if you are going to handle it.

**Comment by haszari** [comment]
@zmaglica yes we have this on our [WCPay Subscriptions maintenance board](https://github.com/orgs/Automattic/projects/403).

**Comment by frosso** [comment]
Closing — WCPay Subscriptions (Stripe Billing) was deprecated in Feb 2025. This Stripe Billing invoice behavior change is no longer applicable.
